index.html

```diff
@@ -311,31 +311,33 @@ <h2 id="heading">About me</h2>
       <ul id="second-media" class="flexs">
         <li>
           <a class="icons" href="https://github.com/leehaney254"
-            ><img src="./assets/gitfoot.png" alt="Github link"
+            ><img src="./assets/gitfoot.png" alt="Link to my githb page"
           /></a>
         </li>
         <li>
           <a
             class="icons"
             href="https://www.linkedin.com/in/leehaney-george-0a4a51178/"
-            ><img src="./assets/linkedinfoot.png" alt="Linkedin icon"
+            ><img
+              src="./assets/linkedinfoot.png"
+              alt="Link to my linkedin page"
           /></a>
         </li>
         <li>
           <a
             class="icons"
             href="https://app.slack.com/client/T47CT8XPG/D0487PRDM7F/rimeto_profile/U03T2ES9771"
-            ><img src="./assets/slackfoot.png" alt="Slack icon"
+            ><img src="./assets/slackfoot.png" alt="Link to my slack page"
           /></a>
         </li>
         <li>
           <a class="icons" href="https://twitter.com/Lee06785586"
-            ><img src="./assets/twitterfoot.png" alt="Twitter icon"
+            ><img src="./assets/twitterfoot.png" alt="Link to my twitter page"
           /></a>
         </li>
         <li>
           <a class="icons" href="georgeleehaney@gmail.com"
-            ><img src="./assets/gmailfoot.png" alt="gmail icon"
+            ><img src="./assets/gmailfoot.png" alt="link to my gmail page"
```

Comment:
Modified the image alt text to be more descriptive and null for decoration. :100: